<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1131</width>
    <height>669</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../Ressources.qrc">
    <normaloff>:/Icons/Icons/book135.svg</normaloff>:/Icons/Icons/book135.svg</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_2" stretch="6,1">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,2">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QComboBox" name="comboBox">
            <property name="currentIndex">
             <number>2</number>
            </property>
            <item>
             <property name="text">
              <string>Elog</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Instruments Description</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Local Files</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QTreeView" name="treeView"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="MakeText" name="InstDesc" native="true"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTextBrowser" name="textBrowser"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1131</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionconnect"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionconnect">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../Ressources.qrc">
     <normaloff>:/Icons/Icons/EarthServer.svg</normaloff>:/Icons/Icons/EarthServer.svg</iconset>
   </property>
   <property name="text">
    <string>connect</string>
   </property>
   <property name="toolTip">
    <string>Connect to database</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MakeText</class>
   <extends>QWidget</extends>
   <header location="global">maketext.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../Ressources.qrc"/>
 </resources>
 <connections/>
</ui>
